### ISSUE TITLE
Development Environment: Run utilities from ICP_binaries

### ISSUE THREAD

**Issue by DominicOram** [ISISComputingGroup/IBEX#5177]
As a developer I would like both upgrading build utilities and installing the IBEX development environment to be as easy as possible. As of https://github.com/ISISComputingGroup/IBEX/issues/5171 cmake was installed in ICP binaries, making it easier to upgrade and install CMAKE.

This ticket is to do the same with other utilities, namely:
* Perl
* Wix
* Maven
* Git

It may not be possible or desirable to do it with all of them as they will be expecting to be run from other directories.

### Acceptance Criteria
- For each of the above utilities they are either being installed as part of ICP_binaries or a note is written as to why they are not

**Comment by FreddieAkeroyd** [comment]
"make" as well?

**Comment by FreddieAkeroyd** [comment]
Git is listed above, but don't you need git to checkout ibex and be able to run create_icp_binaries? I'm not sure moving git there is a good idea - you could be using git for other projects and i'm not sure tying it to ICP_Binaries will be best for these.

**Comment by FreddieAkeroyd** [comment]
Also I wonder is we should split to ICP_Binaries and ICP_Tools, then we could not deploy ICP_Binaries as now but deploy ICP_Tools (smaller) so we have the tools on an instrument if needed for e.g. a DB rebuild

**Comment by DominicOram** [comment]
> Git is listed above, but don't you need git to checkout ibex and be able to run create_icp_binaries? I'm not sure moving git there is a good idea - you could be using git for other projects and i'm not sure tying it to ICP_Binaries will be best for these.

True, I just copied the list of dependencies on the new starter guide and was going to leave it up to the developer to work out which make sense.

**Comment by JamesKingWork** [comment]
Remember to remove install steps for the dependencies from https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/First-time-installing-and-building-(Windows) as part of this ticket
